### ISSUE TITLE
Can Powershell be used to configure the properties "Block executable files from running unless they meet a prevalence, age, or trusted list criteria."?

### ISSUE THREAD

**Issue by arekfurt** [MicrosoftDocs/windows-itpro-docs#1880]
This document gives us the GUID to enable "Block executable files from running unless they meet a prevalence, age, or trusted list criteria.", and another document gives us an (exceedingly brief) description of what that means.  But *how* can we configure the necessary properties of this rule?
 More specifically and immediately (for my own use), can we use Powershell to do so? The text of this document says "all" the rules can be so enabled using PS, so I'm assuming this one is included. But then how can we set the necessary properties for the rule / the "criteria" that the rule is supposed to enforce using Powershell?  

More generally, documentation on how to enable/configure this rule needs to be expanded considerably. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ef258fb1-a3b3-4d22-3322-bd328649da8b
* Version Independent ID: fa731c6b-4b3f-df8a-cd49-8c27ace862fa
* Content: [Enable ASR rules individually to protect your organization](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-exploit-guard/enable-attack-surface-reduction)
* Content Source: [windows/security/threat-protection/windows-defender-exploit-guard/enable-attack-surface-reduction.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/master/windows/security/threat-protection/windows-defender-exploit-guard/enable-attack-surface-reduction.md)
* Product: **w10**
* GitHub Login: @andreabichsel
* Microsoft Alias: **v-anbic**

**Comment by mypil** [comment]
@officedocsbot assign @mypil

**Comment by j0rt3g4** [comment]
Solved by adding a note #3138 . This rule is owned by MS cloud.

**Comment by illfated** [comment]
@arekfurt : Thank you for pointing out this issue.
Hope you find that it has been dealt with properly.

**Comment by arekfurt** [comment]
It's a welcome clarification. The placement of the statement within the article is a little confusing to me (not sure I see the relation between the info and file & folder exceptions), but I'm sure people will figure it out. Though you may want to give that paragraph one more proofread. :)

Also, re-reading this page I continue to wish you folks would change all these ASR docs to consistently and **accurately** describe which versions of Windows can use these rules. But that's a separate matter to file a separate issue about. :)

In short, thank you for your attention to this question. Your revision does resolve a substantial point of confusion I and others have had.

**Comment by j0rt3g4** [comment]
@arekfurt You can see it now live:
https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-exploit-guard/enable-attack-surface-reduction

Thank you for making documentation better and for reporting this issue 👍

**Comment by j0rt3g4** [comment]
@mypil #3172 Corrected the name of the cloud and add it a related link into it.
